### ISSUE TITLE
Failed to create paged result control value: Bad parameter

### ISSUE THREAD

**Issue by kendertas** [DirectoryTree/LdapRecord#251]
Hi, I have a problem to get all users from LDAP.
Connection is fine.

I just run

`$users = User::all();`

but I get a warning
PHP Warning:  ldap_set_option(): Failed to create paged result control value: Bad parameter to an ldap routine (-9)
and it doesn't work

This is the config I use for connection
```
[
            // Mandatory Configuration Options
            'hosts' => xxx.xxx.xxx.xxx,
            'base_dn' => xxx,
            'username' => xxx,
            'password' => xxx,

            // Optional Configuration Options
            'port' => 389,
            'follow_referrals' => false,
            'use_ssl' => false,
            'use_tls' => false,
            'version' => 3,
            'timeout' => 5,

            // Custom LDAP Options
            'options' => []
];
```

"version": "v2.2.0"

Can you help me?
Thanks

**Comment by stevebauman** [comment]
Hi @kendertas,

I can absolutely help you out.

Does downgrading to v2.1.2 resolve the issue for you? Also, which PHP version are you running?

**Comment by stevebauman** [comment]
I've just pushed a commit that I believe should resolve this issue. Can you try downgrading and then using `dev-master` to see if you still encounter this issue?

**Comment by stevebauman** [comment]
Scratch that -- I'm able to reproduce this. Patching now, thanks!

**Comment by stevebauman** [comment]
Released version [v2.2.1](https://github.com/DirectoryTree/LdapRecord/releases/tag/v2.2.1). You're all set!

Let me know if you encounter any further issues 👍

**Comment by kendertas** [comment]
Thanks, it works!
